frontend/src/authenticated/AdminPanel.js

```diff
@@ -1,18 +1,47 @@
-import React from "react";
-import Logout from "./Logout";
+import React, { useState } from "react";
 import { Link } from "react-router-dom";
+import Logout from "./Logout";
+import StyledAdminPanel from "../styles/StyledAdminPanel";
+import useUser from "../utils/customHooks/useUser";
+import chevron from "../assets/chevron.png";
 
 const AdminPanel = () => {
+  const [isHidden, setIsHidden] = useState(true);
+  const [backgroundClass, setBackgroundClass] = useState("");
+  const [imgRotate, setImgRotate] = useState("");
+  const user = useUser();
+
+  const toggleDropdown = () => {
+    setIsHidden(!isHidden);
+    setBackgroundClass(backgroundClass === "bg-light" ? "" : "bg-light");
+    setImgRotate(imgRotate === "img-rotate" ? "" : "img-rotate");
+  };
+
   return (
-    <div className="admin-panel">
-      <Logout />
-      <Link to="/add-post">
-        <p>add post</p>
-      </Link>
-      <Link to="/profile">
-        <p>profile</p>
+    <StyledAdminPanel>
+      <Link to="/add-post" className="add-post-btn">
+        ADD POST
       </Link>
-    </div>
+      <div className={`${backgroundClass} profile`} onClick={toggleDropdown}>
+        <div className="user-info">
+          <img src={user.image} alt="user-img" />
+          <p>
+            {user.firstName} {user.lastName}
+          </p>
+          <img src={chevron} alt="chevron" className={`${imgRotate} chevron`} />
+        </div>
+        <div
+          className={`${
```

Comment:
use classNames package or move that logic to variable
https://www.npmjs.com/package/classnames